### DISCUSSION TITLE
Azure file Share AccessTier

### DISCUSSION THREAD

**Discussion by dbeaupre** [Azure/ResourceModules#3188] [Q&A] [answered]
Hi, 

It's me or it's not possible to set the AccessTier of a fileshare using the actual storage account module version ? I dont see any accessTier property... Thanks

**Comment by eriqua** [comment, marked as answer]
Hi @dbeaupre , thanks for the feedback and for opening this discussion. 

I'm converting it to a Feature Request issue in order for the team to be able to triage it.

Please refer to issue https://github.com/Azure/ResourceModules/issues/3256 going forward.
